include/dab/select_cus_vcvc.h

```diff
@@ -0,0 +1,56 @@
+/* -*- c++ -*- */
+/* 
+ * Copyright 2017 <+YOU OR YOUR COMPANY+>.
+ * 
+ * This is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ * 
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this software; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+
+#ifndef INCLUDED_DAB_SELECT_CUS_VCVC_H
+#define INCLUDED_DAB_SELECT_CUS_VCVC_H
+
+#include <dab/api.h>
+#include <gnuradio/block.h>
+
+namespace gr {
+  namespace dab {
+
+    /*!
+     * \brief <+description of block+>
```

Comment:
missing description.

I'm not expecting much docs here, but at least a definition of what CUS stands for _might_ be a good idea.